pkgs/servers/web-apps/searx/default.nix

```diff
@@ -31,7 +37,7 @@ toPythonModule (buildPythonApplication rec {
   ];
 
   preCheck = ''
-    rm tests/test_robot.py # A variable that is imported is commented out
+    #rm tests/test_robot.py # A variable that is imported is commented out
```

Comment:
Please remove this if it is no longer required.

Reply:
Whoops... missed that in the cleanup.